docs/src/managing-packages.md

```diff
@@ -351,6 +351,15 @@ This message means that a package named `D` has a version conflict.
 Even if you have never `add`ed `D` directly, this kind of error can arise
 if `D` is required by other packages that you are trying to use.
 
+!!! note
+    When tackling these conflicts, first consider that the bigger a project gets, the more likely this is to happen.
+    Using targeted projects for a given task is highly recommended, and removing unused dependencies is a good first
+    step when hitting these issues.
+    For instance, a common pitfall is having more than a few packages in your default (i.e. `(@1.7)`) environment,
```

Comment:
```suggestion
    For instance, a common pitfall is having more than a few packages in your default (i.e. `(@$(Base.VERSION.minor).$(Base.VERSION.minor))`) environment,
```

Reply:
Huh, I didn't know this worked in docs

Reply:
Oh, I just realized that this is a Markdown file; I thought this was a docstring.

It would definitely work inside a docstring, but I'm not sure how to get it to work inside a Markdown file. @mortenpi or @fredrikekre would know.

It definitely would be nice to avoid hardcoding Julia versions in the docs.

Reply:
If this was a block (e.g. a whole paragraph) then you could use an at-eval block, but unfortunately we can't really interpolate into an inline context like this. The best you could do would be something like 

````
using Markdown
```@eval
md"""
!!! note
    The whole admonition with interpolation $(string(Base.VERSION.minor))...
"""
````

Note: it would need the `string()` since the fields are unsigned.

Reply:
Damn. @IanButterworth Want to revert my suggestion, since it won't work correctly?